packages/fluxible-router/lib/createNavLinkComponent.js

```diff
@@ -1,13 +1,14 @@
 /**
- * Copyright 2015, Yahoo! Inc.
+ * Copyright 2015-Present, Yahoo! Inc.
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
-/*global window */
+/*global window,process */
 'use strict';
 var React = require('react');
 var RouteStore = require('./RouteStore');
 var debug = require('debug')('NavLink');
 var navigateAction = require('./navigateAction');
+var __DEV__ = process.env.NODE_ENV !== 'production';
```

Comment:
ha.  Stole it from [fluxible](https://github.com/yahoo/fluxible/blob/master/packages/fluxible/lib/Fluxible.js#L11).  Dont think it is inlined there.

webpack will replace here as well.  And I thought we had concerns about accessing `process.env.NODE_ENV` on every invocation of a hot function (like NavLink render).  Maybe that is no longer a concern?